src/api/pipelineHelpers.ts

```diff
@@ -16,7 +17,7 @@ export const formsToTektonResources = (
   namespace: string,
 ): WizardTektonResources => {
   const { sourceNamespace, sourceApiSecret } = forms.sourceClusterProject.values;
-  const { pipelineName, startImmediately } = forms.pipelineSettings.values;
+  const { pipelineName } = forms.pipelineSettings.values;
```

Comment:
I still don't personally believe we need to make this configurable, but :man_shrugging:

Reply:
Easy to remove if we want to bring it up again. If there is any use case at all for running our wizard more than once I think we need to be able to distinguish the pipeline(run)s created by your wizard run and a previous one.

Maybe a compromise here could just be to have the default value be the name of the source namespace?